envoy-control-source-consul/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/consul/services/ConsulServiceChanges.kt

```diff
@@ -222,3 +237,11 @@ class ConsulServiceChanges(
         }
     }
 }
+
+interface ServiceWatchPolicy {
+    fun shouldBeWatched(service: String, tags: List<String>): Boolean
+}
+
+object NoOpServiceWatchPolicy: ServiceWatchPolicy {
+    override fun shouldBeWatched(service: String, tags: List<String>): Boolean = true
```

Comment:
I'd like to have tags configurable in properties

Reply:
Duplicated by above comment from @Ferdudas97